app/views/nodes/_info.html.erb

```diff
@@ -16,7 +16,7 @@
                   <% facts.each do |fact| %>
                     <dt><%= fact %></dt>
                     <dd>
-                      <% if value = @node.facts[fact.sub(/^facts\./, '')] %>
+                      <% if value = @node.facts[fact] %>
```

Comment:
I am not sure the facts returned by the facts API endpoint actually include the `facts.` prefix in their name. At least in the examples on this page here they do not: https://www.puppet.com/docs/puppetdb/8/api/query/v4/facts